src/libsync/CMakeLists.txt

```diff
@@ -69,31 +69,15 @@ endif()
 
 # These headers are installed for libowncloudsync to be used by 3rd party apps
 set(owncloudsync_HEADERS
-    account.h
-    syncengine.h
-    configfile.h
-    networkjobs.h
-    progressdispatcher.h
-    syncfileitem.h
-    syncresult.h
+    ${CMAKE_CURRENT_BINARY_DIR}/owncloudlib.h
+    logger.h
 )
 
-set(creds_HEADERS
-    creds/abstractcredentials.h
-    creds/httpcredentials.h
+INSTALL(
+    FILES ${owncloudsync_HEADERS}
+    DESTINATION ${INCLUDE_INSTALL_DIR}/${synclib_NAME}
```

Comment:
You should probably add these to the target, then use `install(TARGETS owncloudsync [PUBLIC|PRIVATE]_HEADER DESTINATION <...>)`.

Reply:
As far as I understand PUBLIC_HEADER is a mac os framework thingy.